src/main/java/de/tum/in/www1/artemis/web/rest/TextSubmissionResource.java

```diff
@@ -214,4 +213,77 @@ private void hideDetails(@RequestBody TextSubmission textSubmission) {
                 .collect(Collectors.toList())
         );
     }
+
+    /**
+     * GET  /text-submissions-assessed-by-tutor : get all the textSubmissions assessed by the current tutor.
+     *
+     * @return the ResponseEntity with status 200 (OK) and the list of textSubmissions in body
+     */
+    @GetMapping(value = "/exercises/{exerciseId}/text-submissions-assessed-by-tutor")
```

Comment:
The URL is a bit too long in my opinion. Please rather merge this request with the other two and put the option "assessedByTutor" into a PathVariable